channeldb/kvdb/backend.go

```diff
@@ -0,0 +1,91 @@
+package kvdb
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/lightningnetwork/lnd/channeldb/kvdb/etcd"
+)
+
+// getEnvBackendConfig reads LND_TEST_DB_BACKEND env variable and
+// returns its value if it equals to "etcd" or "bdb" otherwise.
+func getEnvBackendConfig() string {
+	backend, _ := os.LookupEnv("LND_TEST_DB_BACKEND")
```

Comment:
We should switch this over to using a build tag before this lands. 

One way to do this would be something like the following: 
```go
// +build !etcd-db
const TestBackend = "bdb"
```
Then we have a file with the opposite that's gated on specifying that build flag: 
```go
// +build etcd-db
const TestBackend = "etcd"
```

Reply:
ptal